api/v1/openapi.yaml

```diff
@@ -800,6 +800,27 @@ paths:
           description: Invalid request (error parsing parameters)
           schema:
             "$ref": "#/definitions/Error"
+  "/ip/cache/":
```

Comment:
Although we internally call this the "IP cache", do we want to also name it this way in the API? The "cache" part is a bit questionable for some parts of its information, for example CIDR identities, where this is actually the source of truth for this agent's store of this information.

More generally this is "information about an IP" where the IP is the key, then the value includes a bunch of information - identity, encryption key id, k8s pods/namespaces, etc.

Should it just be `/ip/`?